app.js

```diff
@@ -0,0 +1,107 @@
+const TASKS_KEY_NAME = "tasks";
+
+function addTask(event) {
+  event.preventDefault(); // prevent form submission
+  const data = collectDataFromForm();
+  const tasksJSON = localStorage.getItem(TASKS_KEY_NAME);
+  const taskId = tasksJSON ? JSON.parse(tasksJSON).length : 0;
+  const newTR = generateTR(data, taskId);
+  injectTRToDOM(newTR);
+  saveTaskToLocalStorage(data);
+  resetForm();
+}
+function resetForm() {
+  document.getElementById("todo-input").value = ` `;
+  document.getElementById("date-applied").value = ` `;
+  document.getElementById("time-appliedHours").value = ` `;
+  document.getElementById("time-appliedMinutes").value = ` `;
+
+
+}
+
+function collectDataFromForm() {
+  const input = document.getElementById("todo-input").value;
+  const dateApplied = document.getElementById("date-applied").value;
+  const timeAppliedHours = document.getElementById("time-appliedHours").value;
+  const timeAppliedMinutes = document.getElementById("time-appliedMinutes").value;
+
+  return {
+    input,
+    dateApplied,
+    timeAppliedHours,
+    timeAppliedMinutes,
+  };
+}
+
+
+
+
+//Can do time function which converts am/pm to 24hr
+//can do time function to make hours appear as double numbered
+function generateTR(data, id) {
+  // deconstruction
+  const { input, timeAppliedHours, dateApplied, timeAppliedMinutes } = data;
+  let newDate = dateApplied;
+  const dd = dateApplied.slice(8, 10);
+  const mm = dateApplied.slice(5, 7);
+  const yyyy = dateApplied.slice(0, 4);
+  newDate = `${dd}/${mm}/${yyyy}`;
+  let newHour = timeAppliedHours.toString().padStart(2,'0');
+  let newMinute = timeAppliedMinutes.toString().padStart(2,'0');
+  const newTR = `
+        <div class="scrollBox">
```

Comment:
this class applies animation indifferently, whereas it should have applied it only to new notes. If I refresh the page, existing notes show with the animation effect